test/acsets/CSetDataStructures.jl

```diff
@@ -242,7 +241,7 @@ for (dgram_maker, ldgram_maker) in dgram_makers
 
   # Allow type inheritance for data attributes.
   d_abs = dgram_maker(Number)
-  add_parts!(d_abs, :X, 2, height=[10.0, 4])
+  add_parts!(d_abs, :X, 2, height=Number[10, 4])
```

Comment:
Is this a varacsets thing or a general problem with our acsets API? It's not in the spirit of Julia to be so strict about the types. A vector of integers should be implicitly treated as vector of numbers.

Reply:
I reverted the change and there was no error, so this is no longer a problem!